js/web/lib/wasm/jsep/webgpu/program-manager.ts

```diff
@@ -103,17 +106,33 @@ export class ProgramManager {
         }
 
         this.backend.gpuDataManager.release(syncData.id);
-        let inputShapes = '';
-        inputsTensorView.forEach((value, i) => {
-          inputShapes += `input[${i}]: [${value.dims}] | ${tensorDataTypeEnumToString(value.dataType)}, `;
-        });
-        let outputShapes = '';
-        buildArtifact.programInfo.outputs.forEach((value, i) => {
-          outputShapes += `output[${i}]: [${value.dims}] | ${tensorDataTypeEnumToString(value.dataType)}, `;
-        });
-        // eslint-disable-next-line no-console
-        console.log(`[profiling] kernel "${kernelId}|${kernelName}" ${inputShapes}${outputShapes}execution time: ${
-            endTime - startTime} ns`);
+        if (this.backend.env.webgpu.profiling?.ondata) {
```

Comment:
Default startTime is always 0.  And any use case for profilingTimeBase?
``` 
       if (typeof this.backend.profilingTimeBase === 'undefined') {
          this.backend.profilingTimeBase = startTimeU64;
        }
        const startTime = Number(startTimeU64 - this.backend.profilingTimeBase);
        const endTime = Number(endTimeU64 - this.backend.profilingTimeBase);
```

Reply:
`backend.profilingTimeBase` is designed to log the time of the start time of the first completed kernel (`startTime64`). By using this, the startTime and endTime can be relatively small to fit into int53.